pkg/reconciler/openshift/tektonaddon/transformer.go

```diff
@@ -247,3 +249,28 @@ func replacePACTriggerTemplateImages(stepsImages map[string]string) mf.Transform
 		return nil
 	}
 }
+
+func replaceCronjobServiceAccount(serviceAccount string) mf.Transformer {
```

Comment:
if we can remove this after 1.8, then could you move this to a separate file. Then it will be an easy task for anyone to handle.

Reply:
the pr is on 0.55.x :)

Reply:
oh, i see 👍